server/interfaces/interfaces.go

```diff
@@ -272,6 +272,11 @@ type DBHandle interface {
 	IsDeadlockError(err error) bool
 }
 
+type ClickHouseDBHandle interface {
```

Comment:
Is our implementation strongly tied to Clickhouse or can we theoretically switch to a different column DB? Would it make sense to call this OLAPDBHandle?